config/scheduler/settings.ini

```diff
@@ -176,15 +176,15 @@ pa.scheduler.auth.policy.refreshperiod.seconds=30
 
 # The same for the GLOBAL space. The GLOBAL space is shared between each users and each jobs.
 # (concerning the syntax, see above explanations in the INPUT SPACE section)
-#pa.scheduler.dataspace.defaultglobal.url=
+pa.scheduler.dataspace.defaultglobal.url=s3://activeeon-demo-s3
```

Comment:
We cannot define the S3 url inside the scheduling project. I would mean that all releases we produce will use this as global space